EIPS/eip-2274.md

```diff
@@ -6,23 +6,24 @@ discussions-to: https://ethereum-magicians.org/t/eip-2274-multiple-addresses-ran
 status: Draft
 type: Standards Track
 category: Core
-requires: EIP-1109
+requires: 1109
+replaces: 1352
```

Comment:
1352 has a very different purpose so I can't see how this replaces it.

Reply:
That's right @axic my bad. I'll update the document accordingly. Thanks